addons/web/static/tests/views/form/form_view_tests.js

```diff
@@ -3021,7 +3022,6 @@ QUnit.module("Views", (hooks) => {
             "post",
             "read", // reload (successfully clicked on p)
             "some_method",
-            "read", // reload (unsuccessfully clicked on s)
```

Comment:
? looks wrong

Reply:
doActionButton is overwrite in the test. So we don t trigger the retore